macros/src/codegen.rs

```diff
@@ -356,6 +370,10 @@ pub fn generate_code(sm: &ParsedStateMachine) -> proc_macro2::TokenStream {
                             .zip(out_states.iter().zip(guard_action_parameters.iter().zip(guard_action_ref_parameters.iter()))),
                     )
                     .map(|(guard, (action, (out_state, (g_a_param, g_a_ref_param))))| {
+                        let out_state_string = &out_state.to_string()[0..out_state.to_string().find('(').unwrap_or_else(|| out_state.to_string().len())];
```

Comment:
I think a [`split`](https://doc.rust-lang.org/std/primitive.str.html#method.split) is more appropriate and canonical here

Reply:
Fixed. I think it is now more readable than I had it first, thanks.